packages/docusaurus/bin/docusaurus.js

```diff
@@ -94,6 +95,28 @@ program
     });
   });
 
+program
+  .command('info')
+  .description('Shows debugging information about the local environment')
+  .action(() => {
+    console.log(chalk.bold('\nEnvironment Info:'));
+    envinfo
+      .run(
+        {
+          System: ['OS', 'CPU'],
+          Binaries: ['Node', 'Yarn', 'npm'],
+          Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+          npmGlobalPackages: ['@docusaurus'],
```

Comment:
I don't think we should use npmGlobalPackages. Most user will install it as local packages

Can we also use glob ? so anything like `@docusaurus/core`, `@docusaurus/plugin` will be caught